app/helpers/application_helper.rb

```diff
@@ -406,4 +406,35 @@ def preloaded_json
     return '{}' if @preloaded.blank?
     @preloaded.transform_values { |value| escape_unicode(value) }.to_json
   end
+
+  def client_side_setup_data
+    service_worker_url = Rails.env.development? ? 'service-worker.js' : Rails.application.assets_manifest.assets['service-worker.js']
+    current_hostname_without_port = RailsMultisite::ConnectionManagement.current_hostname.sub(/:[\d]*$/, '')
```

Comment:
I would just name the variable `base_url` as well

Reply:
Personally I think this explains what the regex does better than reading `/:[\d]*$/` every time (my regex is rusty and I actually dug out the original commit to make sure this regex wants to strip port ...)

Reply:
OK :+1: